README.md

```diff
@@ -0,0 +1,8 @@
+# PagerDuty Short Circuiter
+This project contains PagerDuty CLI to reduce the time taken from when SRE receives a PD alert to when troubleshooting on the cluster actually begins.
+
+### Activating shell completions
+Run the following to see instructions for various shells:
+`$ pdcli completion --help`
```

Comment:
README.md with steps to set up, build from source and test app

Reply:
- description of the project with google doc link and epic jira link
- instructions to build from source for someone new, exactly similar to backplane-cli 
- add maintainers section, add your names & emails + pooja & dom + mention slack channel (sd-sre-platform) to find us